docs/upgrades/locks/l0ckbox.mdx

```diff
@@ -28,6 +28,24 @@ To unlock, please load the appropriate k3y: tvfkyq
   <summary>Spoilers for l0ckbox's solutions</summary>
 
 ```
+>>user.loc {}
+To unlock, please load the appropriate k3y: tvfkyq
+
+>>sys.upgrades
+...
+017 tier_1 tool         k3y_v1
+...
+
+>>sys.upgrades (i:17, full:true)
```

Comment:
```suggestion
>>sys.upgrades {i:17, full:true}
```
Oops, just saw this as I was about to merge. Sorry I didn't catch it the first time.